Formula/python@3.11.rb

```diff
@@ -419,22 +420,24 @@ def sitecustomize
           sys.path.extend(library_packages)
           # the Cellar site-packages is a symlink to the HOMEBREW_PREFIX
           # site_packages; prefer the shorter paths
-          long_prefix = re.compile(r'#{rack}/[0-9\._abrc]+/Frameworks/Python\.framework/Versions/#{version.major_minor}/lib/python#{version.major_minor}/site-packages')
+          long_prefix = re.compile(r'#{rack}/[0-9\\._abrc]+/Frameworks/Python\\.framework/Versions/#{version.major_minor}/lib/python#{version.major_minor}/site-packages')
```

Comment:
Is this change correct?

If you want to escape `.`, with `r-string`, shouldn't it just be `\.` rather than `\\.`?

```python
>>> print('[0-9\._abrc]')
[0-9\._abrc]
>>> print(r'[0-9\._abrc]')
[0-9\._abrc]
>>> print(r'[0-9\\._abrc]')
[0-9\\._abrc]
```